types/database.ts

```diff
@@ -1,40 +1,40 @@
-import { Document } from 'mongoose';
+import {Document} from 'mongoose'
 
 export interface IUser extends Document {
```

Comment:
db dan hammani language ni uzga to'g'illab qo'yiw kere

Reply:
dbdagi lang umuman ishlatilmidiku endi

Reply:
to'ri etasiz kalla qo'ydim, unaqada uyodan umuman o'cirvoriw kere

Reply:
fixed